app/controllers/v1/sessions_controller.rb

```diff
@@ -25,15 +25,23 @@ def new
 
       StatsD.increment(STATSD_SSO_NEW_KEY, tags: ["context:#{type}"])
       url = url_service.send("#{type}_url")
+
       if type == 'slo'
         Rails.logger.info('SSO: LOGOUT', sso_logging_info)
         reset_session
+      elsif type == 'ssoe_slo'
+        Rails.logger.info('SSOe: LOGOUT', sso_logging_info)
+        reset_session
```

Comment:
```suggestion
if type == 'slo' || type == 'ssoe_slo'
  Rails.logger.info("LOGOUT of type #{type}", sso_logging_info)
  reset_session
end
```

I think we can DRY this up a bit.

Reply:
you're right—I think the logs are just used for debugging so it should be fine to change the message and DRY here. Thanks for the suggestion